EIPS/eip-1679.md

```diff
@@ -18,6 +18,10 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - Activation: TBD
 - Included EIPs: TBD
 
+## Proposed changes
```

Comment:
Can you rename this to "Proposed Changes"? It seems that is the styling we are going for. Also please rebase or something to make the automerger happy.